src/commons/sagas/GitHubPersistenceSaga.ts

```diff
@@ -66,22 +66,20 @@ function* githubLogoutSaga() {
 }
 
 function* githubOpenFile(): any {
-  const octokit = GitHubUtils.getGitHubOctokitInstance() || {
-    users: { getAuthenticated: () => {} },
-    repos: { listForAuthenticatedUser: () => {} }
-  };
-
-  type ListForAuthenticatedUserResponse = GetResponseTypeFromEndpointMethod<
-    typeof octokit.repos.listForAuthenticatedUser
-  >;
-  const results: ListForAuthenticatedUserResponse = yield call(
-    octokit.repos.listForAuthenticatedUser
-  );
+  const octokit = GitHubUtils.getGitHubOctokitInstance();
```

Comment:
Ideally, the return type of `GitHubUtils.getGitHubOctokitInstance()` should be `Octokit | undefined` instead of `any` but changing the type will result in having to fix a lot of type errors, which I deem to be out of scope of this PR.